src/screens/SendSynthetics/SendSyntheticsSelect.js

```diff
@@ -0,0 +1,373 @@
+            template: SelectorInputTemplate,
+            config: {
+              label: 'Synthetic asset to send',
+              hasInput: true,
+              options: [],
+              horizontalOptions: [],
+              placeholderSelector: 'select',
+              placeholderInput: '0',
+              inputRef: (ref) => { this.fromInputRef = ref; },
+            },
+            transformer: {
+              parse: (value) => {
+                let formattedAmount = value.input;
+                if (value.input) formattedAmount = value.input.toString().replace(/,/g, '.');
+                return { ...value, input: formattedAmount };
+              },
+              format: (value) => {
+                let formattedAmount = value.input;
+                if (value.input) formattedAmount = value.input.toString().replace(/,/g, '.');
+                return { ...value, input: formattedAmount };
+              },
+            },
+          },
+        },
+      },
+    };
+    this.handleFormChange = debounce(this.handleFormChange, 500);
+  }
+
+  componentDidMount() {
+    this.generateSupportedAssetsOptions();
+    this.props.initSyntheticsService();
+  }
+
+  generateSupportedAssetsOptions = () => {
+    const { supportedAssets } = this.props;
+    const assets = [...supportedAssets] // prevent mutation of param
+      .sort((a, b) => a.symbol.localeCompare(b.symbol))
+      .map(({ symbol, iconUrl, ...rest }) => ({
+        key: symbol,
+        value: symbol,
+        icon: iconUrl,
+        iconUrl,
+        symbol,
+        ...rest,
+      }));
+    const initialFormState = {
+      ...this.state.value,
+      fromInput: {
+        selector: assets.find(({ symbol }) => symbol === ETH) || {},
+        input: '',
+      },
+    };
+    const thisStateFormOptionsCopy = { ...this.state.formOptions };
+    thisStateFormOptionsCopy.fields.fromInput.config.options = assets;
+    this.setState({ formOptions: thisStateFormOptionsCopy, value: initialFormState });
+  };
+
+  handleFormChange = (value: Object) => {
+    const { intentError } = this.state;
+    let updatedState = { value };
+    if (intentError) updatedState = { ...updatedState, intentError: null };
+    this.setState(updatedState);
+    if (this.syntheticsForm) this.syntheticsForm.getValue(); // validates form
+  };
+
+  handleFormSubmit = () => {
+    const { submitPressed, value } = this.state;
+    if (submitPressed) return;
+    this.setState({ submitPressed: true, intentError: null }, () => {
+      const validation = this.syntheticsForm.validate();
+      const { errors = [] } = validation;
+      const assetCode = get(value, 'fromInput.selector.symbol');
+      if (errors.length || !assetCode) {
+        this.setState({ submitPressed: false });
+        return;
+      }
+      const amount = parseNumericAmount(value);
+      const { navigation, assets, supportedAssets } = this.props;
+      const assetsData = Object.keys(assets).map(id => assets[id]);
```

Comment:
Use getAssetsAsList()

Reply:
This branch didn't have it yet, will update